sys/i386/include/cpufunc.h

```diff
@@ -147,43 +146,37 @@ cpuid_count(u_int ax, u_int cx, u_int *p)
 static __inline void
 enable_intr(void)
 {
-
 	__asm __volatile("sti");
 }
 
 static __inline void
 cpu_monitor(const void *addr, u_long extensions, u_int hints)
 {
-
 	__asm __volatile("monitor"
 	    : : "a" (addr), "c" (extensions), "d" (hints));
 }
 
 static __inline void
 cpu_mwait(u_long extensions, u_int hints)
 {
-
 	__asm __volatile("mwait" : : "a" (hints), "c" (extensions));
 }
 
 static __inline void
 lfence(void)
 {
-
 	__asm __volatile("lfence" : : : "memory");
 }
 
 static __inline void
 mfence(void)
 {
-
 	__asm __volatile("mfence" : : : "memory");
 }
 
 static __inline void
 sfence(void)
 {
-
```

Comment:
This seems unrelated, but the change itself is good.

Reply:
This dates to a prior effort to merge these two files and I kept it because it's a basically sensible change so didn't see a need to throw it out.